vitals/BoundryConditions.java

```diff
@@ -0,0 +1,61 @@
+package vitals;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+public class BoundryConditions {
+    static Map<String, Map<String, String>> conditions;
+
+    public static void setConditions() {
+        conditions = new HashMap<>();
+        Map<String, String> soc = new HashMap<>();
+        soc.put("0-20", "LOW_SOC_BREACH");
```

Comment:
The values are hardcoded. Ideally once we know the low and high values along with tolerance (this can be 5, 10 or 15 %) the program should be able to calculate all required values.
Hardcoding should be reduced as much as possible.